src/shared/defaultAwsContext.ts

```diff
@@ -56,8 +57,8 @@ export class DefaultAwsContext implements AwsContext {
         return this.currentCredentials?.accountId
     }
 
-    public getCredentialDefaultRegion(): string | undefined {
-        return this.currentCredentials?.defaultRegion
+    public getCredentialDefaultRegion(): string {
+        return this.currentCredentials?.defaultRegion ?? DEFAULT_REGION
```

Comment:
I'm thinking we should remove DEFAULT_REGION entirely. Everywhere we use DEFAULT_REGION, should instead use `getCredentialDefaultRegion()`.